pkg/README.md

```diff
@@ -6,29 +6,30 @@ This quickstart guide is intended for engineers familiar with k8s and model serv
  - Envoy Gateway [v1.2.1](https://gateway.envoyproxy.io/docs/install/install-yaml/#install-with-yaml) or higher
  - A cluster that has built-in support for `ServiceType=LoadBalancer`. (This can be validated by ensuring your Envoy Gateway is up and running)
    - For example, with Kind, you can follow these steps: https://kind.sigs.k8s.io/docs/user/loadbalancer
+- 3 GPUs to run the vLLM deployment. Adjust the number of replicas as needed.
 
 ### Steps
 
-1. **Deploy Sample vLLM Application**
+1. **Install the Inference Extension CRDs:**
 
-   Create a Hugging Face secret to download the model [meta-llama/Llama-2-7b-hf](https://huggingface.co/meta-llama/Llama-2-7b-hf). Ensure that the token grants access to this model. 
-   Deploy a sample vLLM deployment with the proper protocol to work with the LLM Instance Gateway.
-   ```bash
-   kubectl create secret generic hf-token --from-literal=token=$HF_TOKEN # Your Hugging Face Token with access to Llama2
-   kubectl apply -f ./manifests/vllm/vllm-lora-deployment.yaml
+   ```sh
+   kubectl apply -f config/crd/bases
    ```
 
-1. **Install the CRDs into the cluster:**
+1. **Deploy Sample vLLM Application**
+
+   Create a Hugging Face secret to download the model [meta-llama/Llama-2-7b-hf](https://huggingface.co/meta-llama/Llama-2-7b-hf). Ensure that the token grants access to this model.
 
-   ```sh
-   kubectl apply -f config/crd/bases
+   Replace `$HF_TOKEN` in `./manifests/vllm/deployment.yaml` with your Hugging Face secret and then deploy the sample vLLM deployment.
+   ```bash
+   kubectl apply -f ./manifests/vllm/deployment.yaml
    ```
 
-1. **Deploy InferenceModel and InferencePool**
+1. **Deploy InferenceModel**
 
-   Deploy a sample InferenceModel and InferencePool configuration based on the vLLM deployments mentioned above.
+   Deploy a sample InferenceModel configuration based on the vLLM deployments mentioned above.
    ```bash
-   kubectl apply -f ./manifests/inferencepool-with-model.yaml
+   kubectl apply -f ./manifests/inferencemodel.yaml
    ```
```

Comment:
Note that the InferencePool CR is now part of the ext-proxy deployment. The InferenceModel had to be split from the InferencePool since e2e tests will create InferenceModels for each test case. The InferenceModel could have its own manifest but IMHO it makes sense to bundle with the ext-proc since the ext-proc is specifically configured for this InferencePool.